.github/workflows/kind-e2e.yaml

```diff
@@ -124,10 +125,11 @@ jobs:
           namespace-resources: virtualservices
           enable-tls: 1
 
-        - ingress: istio-ambient
-          namespace-resources: virtualservices
-          ingress-class: istio
-          ambient: 1
+        # Disabled due to flakiness: https://github.com/knative/serving/issues/14637
+        # - ingress: istio-ambient
+        #   namespace-resources: virtualservices
+        #   ingress-class: istio
+        #   ambient: 1
```

Comment:
linter warning on this file is legit - unsure why it can't detect the `||` and sorta ignore that the object is not defined (cc @kvmware)

Reply:
👍 I also uncommented the variable to make sure this works.